.github/workflows/build.yml

```diff
@@ -36,11 +37,6 @@ jobs:
           with:
             submodules: recursive
 
-        - name: "Set up Java: 11"
```

Comment:
Note, JDK 11 is preinstalled on all the images we use.  We don't need to install it separately.